tests/assets/provisioning.yml

```diff
@@ -37,3 +35,7 @@
   site: "digitall"
 
 - executeScript: "script-01-create-categories.groovy"
+
+- uninstallBundle: 
+    - "org.jahia.modules/content-editor/4.6.0.SNAPSHOT"
+    - "org.jahia.modules/jcontent/2.12.0.SNAPSHOT"
```

Comment:
you can probably omit the version here

Reply:
thanks. remove it from CE. also I don't think we need to uninstall jcontent since it gets updated anyways.